### PR TITLE
wireshark: enable sshdump, ciscodump

### PR DESCRIPTION
**Pull request by purpurrot**
enables sshdump utility of wireshark
https://www.wireshark.org/docs/man-pages/sshdump.html

### CODE REVIEW

File: srcpkgs/wireshark/template [hunk 1]
```diff
@@ -6,8 +6,9 @@ build_style=cmake
 hostmakedepends="pkg-config flex perl python3-devel mit-krb5-devel qt5-tools
  qt5-host-tools qt5-qmake"
 makedepends="libpcap-devel zlib-devel lua52-devel mit-krb5-devel gnutls-devel
- libmaxminddb-devel libgcrypt-devel libcap-devel sbc-devel c-ares-devel
- qt5-devel qt5-tools-devel qt5-multimedia-devel qt5-svg-devel"
+ libmaxminddb-devel libgcrypt-devel libcap-devel sbc-devel c-ares-devel libxml2-devel
+ libssh-devel libnl3-devel spandsp-devel tiff-devel libcap-progs liblz4-devel
```
Comment (jnbr) [inline srcpkgs/wireshark/template]: Is adding `tiff-devel` related to enable sshdump and ciscodump?

**Comment by jnbr** [comment]
you need to increase the revision of the template

File: srcpkgs/wireshark/template (hunk 1 above)
Comment (purpurrot) [reply srcpkgs/wireshark/template]: it is needed by spandsp

/usr/include/spandsp.h:43:10: fatal error: tiffio.h: No such file or directory
 #include <tiffio.h>
          ^~~~~~~~~~
compilation terminated.

**Comment by purpurrot** [comment]
there are some other features that are not enabled because some libraries are not packages for void

https://github.com/google/snappy
https://www.ibr.cs.tu-bs.de/projects/libsmi/download.html
https://github.com/BelledonneCommunications/bcg729

**Comment by maxice8** [comment]
snappy libary is available

File: srcpkgs/wireshark/template (hunk 1 above)
Comment (jnbr) [reply srcpkgs/wireshark/template]: ok

**Pull request merged**